src/com/cft/contactmerge/tests/ContactTest.java

```diff
@@ -1,6 +1,10 @@
 package com.cft.contactmerge.tests;
```

Comment:
There should be a test for each contact part in addition to the contact tests. In other words, a test for Address, Email, Name, and Phone. Some may argue that each part should have an interface so that you can create mocks, especially for the isMatch() method. I'm leaning that way but will leave it up to you as that does add a fair amount of code. It doesn't really add much value for Email and Phone but could make the Contact tests easier for Name and Address.

Reply:
Yes, I will add these in the next pull.